pkg/rpc/compat_drpc.go

```diff
@@ -97,16 +97,6 @@ func (c *Conn) IrreparableInspectorClient() IrreparableInspectorClient {
 	return NewIrreparableInspectorClient(c.raw)
 }
 
-// NewKadInspectorClient returns the drpc version of a KadInspectorClient
```

Comment:
seems unrelated but I guess okay to take out here while you're at it

Reply:
that is auto generated.  i talked to Jennifer about the inspector changes when i ran make proto